src/main/java/Task.java

```diff
@@ -0,0 +1,24 @@
+import java.util.*;
+
+public class Task {
+    public static final String descriptionPrefix = "description: ";
+    private String description;
+    private boolean important;
```

Comment:
Boolean variables/methods should be named to sound like booleans ? 

Reference: https://se-education.org/guides/conventions/java/index.html